apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -0,0 +1,154 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import commonMsg from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+import {
+  studentAnswerDataPropType,
+  questionDataPropType
+} from './assessmentDataShapes';
+
+export const COLUMNS = {
+  QUESTION: 0,
+  STUDENT_ANSWER: 1,
+  CORRECT_ANSWER: 2,
+};
+
+class StudentAssessmentOverviewTable extends Component {
+  static propTypes= {
+    questionAnswerData: PropTypes.arrayOf(questionDataPropType),
```

Comment:
This naming is a little unclear to me - is `questionAnswerData` and array of the possible answer choices? Could just be me, but might be worth comments explicitly clarifying what each of these propTypes represent.

Reply:
The questionAnswerData is the multiple choice data.  Multiple choice data is an array or objects that includes questions, answers etc.